desktop/core/src/desktop/js/apps/notebook2/components/aceEditor/AceLocationHandler.ts

```diff
@@ -755,7 +757,7 @@ export default class AceLocationHandler implements Disposable {
   checkForSyntaxErrors(statementLocation: ParsedLocation, cursorPosition: Ace.Position): void {
     if (
       this.sqlSyntaxWorkerSub &&
-      (this.getDialect() === DIALECT.impala || this.getDialect() === DIALECT.hive)
+      (this.getDialect() === IMPALA_DIALECT || this.getDialect() === HIVE_DIALECT)
```

Comment:
Good to know it is whitelisted here for now until we port to have the other dialects.

How much work to generify it if we add a dialect property `has_syntax_check=tre/false` in
https://github.com/cloudera/hue/blob/master/desktop/core/src/desktop/lib/connectors/types.py#L43

Reply:
Not sure, I bet we can get rid of all the dialect specific conditions in these files quite easily. We know if a syntax parser  exists or not so fairly quick.